website/static/website/style.css

```diff
@@ -64,3 +64,18 @@ body {
 .profile-dropdown .img-responsive {
     display: inline;
 }
+
+.btn-as-link {
+    background: none !important;
+    color: inherit;
+    border: none; 
+    padding: 0 !important;
+    font: inherit;
+    border-bottom: 1px solid #444; 
+    cursor: pointer;
+}
+
+.modal-body {
+    max-height: calc(100vh - 212px);
```

Comment:
212px = height of header and footer together

Reply:
No. The website scales.  If you zoom in, the footer will grow in height because the contents wrap.